zenoh/src/api/query.rs

```diff
@@ -79,27 +79,63 @@ impl Default for QueryConsolidation {
     }
 }
 
-/// Structs returned by a [`get`](Session::get).
+/// Error returned by a [`get`](Session::get).
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct ReplyError {
+    pub(crate) payload: ZBytes,
+    pub(crate) encoding: Encoding,
+}
+
+impl ReplyError {
+    /// Gets the payload of this ReplyError.
+    #[inline]
+    pub fn payload(&self) -> &ZBytes {
+        &self.payload
+    }
+
+    /// Gets the payload of this ReplyError.
+    #[inline]
+    pub fn payload_mut(&mut self) -> &mut ZBytes {
```

Comment:
Is `payload_mut` really necessary?

Reply:
It was copied from `Value`, but I think we can remove it yes, so I will do it.